src/Models/Role.php

```diff
@@ -18,4 +19,13 @@ public function permissions()
     {
         return $this->belongsToMany(Voyager::modelClass('Permission'));
     }
+
+    public static function getRelationship($id)
```

Comment:
This was only added in the `Role` model, but relationships can be used in any model... This is breaking Voyager: #2431

Reply:
This should be added to the base voyager model, or as a trait.

Reply:
Agreed: #2432 2432

Reply:
I have also created one in #2433